webui/src/index.js

```diff
@@ -1,6 +1,7 @@
 import "./main.scss";
 
 import { Main } from './Main.elm'
+import { HelloWorld } from './helloWorld.elm'
```

Comment:
I missed this before. I wonder if we can make this work within our existing elm program rather than a separate elm instance? I suspect having two elm runtimes going explains why we weren't seeing requestAnimationFrame work.

Reply:
I pushed a commit that refactors this, super happy with how this turned out!